src/strategies/otterspace-badges/index.ts

```diff
@@ -75,9 +79,17 @@ function applyBadgeWeights(badges: [], options: any) {
   badges.forEach((badge: any) => {
     const badgeAddress = badge.owner.toLowerCase();
 
-    if (badgeWeights[badgeAddress]) return;
+    const badgeWeight = getBadgeWeight(options.specs, badge.spec.id);
 
-    badgeWeights[badgeAddress] = getBadgeWeight(options.specs, badge.spec.id);
+    // picks the highest weight when multiple badges are held by the same address
+    if (
+      !badgeWeights[badgeAddress] ||
+      badgeWeight > badgeWeights[badgeAddress]
+    ) {
+      badgeWeights[badgeAddress] = badgeWeight;
+    } else {
+      return;
+    }
```

Comment:
Hi @rahulrumalla There is a active proposal on cowgrants.eth space, that is using this space, can this change create a problem in someone's voting power? better to wait until the proposal is closed

Reply:
oh i think it may not cause a problem looking at the code, can confirm? :)

Reply:
It shouldn't cause a problem